### DISCUSSION TITLE
Where is message defined on HTTParty::Response?

### DISCUSSION THREAD

**Discussion by ajtoo** [jnunemaker/httparty#789] [Q&A]
Maybe this is a silly question, but until lately, I haven't thought much of it. When I try to mock the `#message` method for an `instance_double`, it comes back with `HTTParty::Response class does not implement the instance method: message`. If I introspect the method, there doesn't seem to be a source location, and I can't find a documented definition. Is this just a native, non ruby method from somewhere?

```ruby
[12] pry(main)> res.method(:message).source_location
=> nil
[13] pry(main)> res.message
=> "Unauthorized"
[14] pry(main)> res.method(:message)
=> #<Method: HTTParty::Response#message(*)>
[15] pry(main)> res.method(:message).owner
=> HTTParty::Response
```

**Comment by thushalb** [comment]
Did you ever find out how to resolve this? I am trying to use this for a unit test but ran into this issue @ajtoo

**Comment by jnunemaker** [comment]
Message is likely delegated to the original response net http response via
delegation or method missing. I’m away from computer but that is my bet.
